File: Atividades/Lista2/Java/Formas.java
package Atividades.Lista2.Java;

import java.util.Scanner;

public class Formas {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int c1 = input.nextInt();
        int l1 = input.nextInt();
        input.nextLine();
        int c2 = input.nextInt();
        int l2 = input.nextInt();
        input.nextLine();

        Retangulo r1 = new Retangulo();
        Retangulo r2 = new Retangulo();

        r1.setComprimento(c1);
        r1.setLargura(l1);
        r2.setComprimento(c2);
        r2.setLargura(l2);


        System.out.printf("%d %d %d %d\n", r1.getComprimento(), r1.getLargura(), r1.calculaPerimetro(), r1.calculaArea());

        System.out.printf("%d %d %d %d\n", r2.getComprimento(), r2.getLargura(), r2.calculaPerimetro(), r2.calculaArea());

        input.close();
    }
}

class Retangulo {
    private int comprimento = 1;
    private int largura = 1;

    
    public int getComprimento() {
        return comprimento;
    }
    public void setComprimento(int comprimento) {
        if (comprimento > 0 && comprimento < 20)
            this.comprimento = comprimento;
    }
    public int getLargura() {
        return largura;
    }
    public void setLargura(int largura) {
        if (largura > 0 && largura < 20)
            this.largura = largura;
    }

    public int calculaPerimetro() {
        return 2 * (comprimento + largura);
    }

    public int calculaArea() {
        return comprimento * largura;
    }
}

File: Atividades/Lista2/C++/formas.cpp
#include <iostream>

using namespace std;

class Retangulo {
private:
    int comprimento = 1;
    int largura = 1;

public:
    int getComprimento() {
        return comprimento;
    }
    int getLargura() {
        return largura;
    }
    void setComprimento(int comprimento) {
        if (comprimento > 0 && comprimento < 20)
            this->comprimento = comprimento;
    }
    void setLargura(int largura) {
        if (largura > 0 && largura < 20)
            this->largura = largura;
    }
    int calculaPerimetro() {
        return 2 * (comprimento + largura);
    }
    int calculaArea() {
        return comprimento * largura;
    }
};

int main() {
    int c1, l1, c2, l2;
    cin >> c1 >> l1 >> c2 >> l2;
    
    Retangulo r1, r2;

    r1.setComprimento(c1);
    r1.setLargura(l1);
    r2.setComprimento(c2);
    r2.setLargura(l2);

    cout << r1.getComprimento() << " " << r1.getLargura() << " " << r1.calculaPerimetro() << " " << r1.calculaArea() << endl;

    cout << r2.getComprimento() << " " << r2.getLargura() << " " << r2.calculaPerimetro() << " " << r2.calculaArea() << endl;

    return 0;
}